### ISSUE TITLE
Question about the published releases

### ISSUE THREAD

**Issue by nithya16886** [kesha-antonov/react-native-action-cable#17]
Hello,

There is a change to allow headers to be sent in, while creating consumers, that is present in master - https://github.com/kesha-antonov/react-native-action-cable/blob/master/lib/action_cable/consumer.coffee, but not been released.

Is there a plan to release this change? We have a requirement to update one of the request headers and it would be helpful.

Thanks!!

**Comment by kesha-antonov** [comment]
Hi
Sure
I'll do it today/tomorrow

**Comment by kesha-antonov** [comment]
BTW you can also do like this: 

```
ActionCable.createConsumer(`${CONSTANTS.WS_URI}?session_token=${encodeURIComponent(token)}`)
```

**Comment by kesha-antonov** [comment]
Released 1.1.5
Also updated readme: https://github.com/kesha-antonov/react-native-action-cable?tab=readme-ov-file#methods

**Comment by nithya16886** [comment]
Thank you for releasing this!
